### ISSUE TITLE
Create user admin not work. Any error, but setup.php not insert anything on table admin

### ISSUE THREAD

**Issue by abkrim** [postfixadmin/postfixadmin#448]
Ubuntu 20.04

PostfixAdmin 3.3.5
Nginx 1.18.0
PHP 7.4 FPM socket based
Fresh install form source

https://centralpostfix.castris.com/setup.php?debug=1
Setup password (hash in config.local.php )

Put password setup.
Email for admin
New password copy and past en inputs.
log tailed no shoe any problem

After click, any error.
Verify and not work.

Check database, ... oh... any admin in table admin.

![screencapture-centralpostfix-castris-setup-php-2021-01-30-20_04_11](https://user-images.githubusercontent.com/1238625/106365546-9c80c280-6336-11eb-8084-100727a536cd.png)

**Comment by DavidGoodwin** [comment]
Please check your error log(s) - some instructions that may help are here - https://github.com/postfixadmin/postfixadmin/blob/master/DOCUMENTS/FAQ.txt#L38

**Comment by abkrim** [comment]
Hi.

My logs for php are in:

**Comment by abkrim** [comment]
Hi 

My logs not show anything. (correct permissions)

error_log	/var/log/php-fpm/centralpostfix.castris.com-error.log	/var/log/php-fpm/centralpostfix.castris.com-error.log

Nginx not show anything
```sudo tail -f /var/log/php-fpm/centralpostfix.castris.com-error.log /var/log/php7.4-fpm.log  /var/log/syslog /var/log/nginx/domains/centralpostfix.castris.com.error.log | grep -v Fire

(empty after run setup.)
```

Also tried, insert into config.lol.php `ini_set('error_log', '/tmp/postfixadmin.log'); // example only`

Not show anything

Of course restart (not reload) php7.4-fpm, nginx befor try eveiry step.

Best regards

**Comment by DavidGoodwin** [comment]
Do you have apparmor of SELinux installed/operating ? 

What `$CONF['encrypt']` setting are you trying to use?

**Comment by 1RUSUA1** [comment]
Have the same problem!
Centos 7, PHP 7.2.34,  Postfixadmin 3.3.1.
And I have got another one stupid problem - I use MYSQL **Server version: 5.5.62-log MySQL Community Server (GPL) by Remi** 
and on it the query **SELECT password FROM `admin` WHERE username = 'admin@xxxxxxxx' AND active = '1';** returns Empty set, but if I change the last part of the query to **SELECT password FROM `admin` WHERE username = 'admin@delta.kh.ua' AND active='1';** - it works! I have checked that in mysql console. I consider this is on the mysql side problem.

**Comment by abkrim** [comment]
Is "default" installation without touch anything.

Also after upgrade.php
```
$CONF['encrypt'] = 'md5crypt';
```
Mysql 8.0
User postfix admin with native password.

**Comment by abkrim** [comment]
After lost some tiem.

- Try on local not work.
- Try on local with githib master brancha. Work perfectly.
- Remove installation and database on production.
- Install git master branch.
- Reading and configure
- Work perfectly.

Any changes on config of PHP, FPM, MySQL.

Best regards.

**Comment by DavidGoodwin** [comment]
@abkrim the only difference between the postfixadmin_3.3 branch and master should be ... 

 * the dovecot postpassword stuff,  
 * documentation improvements 
 * a slightly improved setup.php which may highlight issues with dovecot / password encryption

I wouldn't have expected any of these to make any difference to whether you can login or not.

**Comment by heliosyne** [comment]
Will there be a 3.3.6 to cover these?

**Comment by abkrim** [comment]
I do not know the changes. Don't take time to review them. I write here, because it seemed important to me.
I have never used postfixadmin. If I had a time limit maybe I would take a break to analyze the problem and see if I could put a PR.
Now, being practical, I needed something fast for a server without a panel that requires a little account management, and I did not want to do it bareback, since it is not for me.
What I am clear about is that I consumed the downloadable neither it worked locally nor in production. Maybe I'm not a good programmer but a systems administrator, and I was able to get around it.
Hope to come back soon, if I like the software.
